package.json

```diff
@@ -110,115 +110,152 @@
     ],
     "configuration": [
       {
-        "title": "%configuration.SAS.session.title.general%",
+        "title": "%configuration.SAS.title%",
+        "order": 0,
+        "id": "connectionProfiles",
         "properties": {
-          "SAS.session.host": {
-            "type": "string",
+          "SAS.output.HTML5": {
             "order": 0,
-            "description": "%configuration.SAS.session.host%"
-          },
-          "SAS.session.outputHtml": {
             "type": "boolean",
             "default": true,
-            "description": "%configuration.SAS.session.outputHtml%"
+            "description": "%configuration.SAS.output.HTML5%"
           },
-          "SAS.session.computeContext": {
-            "type": "string",
-            "default": "SAS Job Execution compute context",
-            "description": "%configuration.SAS.session.computeContext%"
-          }
-        }
-      },
-      {
-        "title": "%configuration.SAS.session.title.password%",
-        "properties": {
-          "SAS.session.user": {
-            "type": "string",
-            "description": "%configuration.SAS.session.user%"
-          },
-          "SAS.session.clientId": {
-            "type": "string",
-            "description": "%configuration.SAS.session.clientId%"
-          },
-          "SAS.session.clientSecret": {
-            "type": "string",
-            "description": "%configuration.SAS.session.clientSecret%"
-          }
-        }
-      },
-      {
-        "title": "%configuration.SAS.session.title.token%",
-        "properties": {
-          "SAS.session.tokenFile": {
-            "type": "string",
-            "description": "%configuration.SAS.session.tokenFile%"
+          "SAS.defineConnectionProfiles": {
+            "order": 1,
+            "type": "object",
+            "description": "%configuration.SAS.defineConnectionProfiles%",
+            "scope": "application",
```

Comment:
Any reason to scope it to application? Looking at VS Code's documentation, I think the default "window" should work. Workspace and remote settings are valid scenarios.

Reply:
Agreed. I have changed the scope to `window`.